fern/generators.yml

```diff
@@ -0,0 +1,39 @@
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-go-sdk
+        version: 0.23.2
```

Comment:
These can go to 0.27.0. You can always see the latest here https://github.com/fern-api/fern?tab=readme-ov-file#sdk-generators